src/service/FlywheelService.ts

```diff
@@ -90,11 +91,22 @@ export class FlywheelService extends APIClient {
      * Return all transactions for a specific phase
      * @param phaseName
      * @param view
+     * @deprecated Because same functionality plus filter possibility can be found in POST method: fetchTransactionsForPhaseWithFilter
      */
     async fetchTransactionsForPhase(phaseName: string, view: string = 'card'): Promise<AxiosResponse> {
         return this.invokeApi(`/transactions/phases/${phaseName}?view=${view}`);
     }
 
+    /**
+     * Return all transactions for a specific phase
+     * @param phaseName
+     * @param view
+     * @param flowdsl
```

Comment:
```suggestion
     * @param Flowdsl flowdsl
```
Das kann helfen, dass IntelliJ da bessere Code-Completion gibt